docs/getting-started-guides/kubeadm.md

```diff
@@ -149,28 +183,12 @@ For example:
 
 A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
 
-**YOUR CLUSTER IS NOT READY YET!**
-
-Before you can deploy applications to it, you need to install a pod network.
-
-### (4/4) Installing a pod network
-
-You must install a pod network add-on so that your pods can communicate with each other when they are on different hosts.
-**It is necessary to do this before you try to deploy any applications to your cluster.**
-
-Several projects provide Kubernetes pod networks.
-You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
-
-By way of example, you can install [Weave Net](https://github.com/weaveworks/weave-kube) by logging in to the master and running:
-
-    # kubectl apply -f https://git.io/weave-kube
-    daemonset "weave-net" created
+### (Optional) Control your cluster from machines other than the master
 
-If you prefer [Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm) or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm), please refer to their respective installation guides.
-You should only install one pod network per cluster.
+In order to get a kubectl on your laptop for example to talk to your cluster, you need to copy the `KubeConfig` file from your master to your laptop like this:
```

Comment:
It does create a `admin.json` kubeconfig file that can be used for accessing the cluster, but not tied to a specific user with permissions